### PR TITLE
calculate container cpi by cycles and instructions

### PR DESCRIPTION
**Pull request by sun-yuliang**
#### What type of PR is this?
Bug fix

#### What this PR does / why we need it:
Calculate container cpi metric by delta(cycles)/delta(instructions).

### CODE REVIEW

File: pkg/metaserver/agent/metric/metric.go [hunk 1]
```diff
@@ -134,6 +134,9 @@ func NewMalachiteMetricsFetcher(emitter metrics.MetricEmitter) MetricsFetcher {
 }
 
 type MalachiteMetricsFetcher struct {
+	cpuCgDataV1 cgroup.CPUCgDataV1
```
Comment (waynepeking348) [inline pkg/metaserver/agent/metric/metric.go:137]: if we store this here, why we still need metricStore? it's wired to just keep cpuCgDataV1/cpuCgDataV2 here, why not get previous Instructions from  metricStore?

Comment (sun-yuliang) [reply pkg/metaserver/agent/metric/metric.go]: fixed

**Review by waynepeking348 — APPROVED** [review]

File: pkg/metaserver/agent/metric/metric.go [hunk 2]
```diff
@@ -584,9 +614,9 @@ func (m *MalachiteMetricsFetcher) processCgroupPerfData(podUID, containerName st
 	if perf == nil {
 		return
 	}
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCPIContainer, perf.Cpi)
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCyclesContainer, perf.Cycles)
-	m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUInstructionsContainer, perf.Instructions)
+	// m.metricStore.SetContainerMetric(podUID, containerName, consts.MetricCPUCPIContainer, perf.Cpi)
```
Comment (cheney-lin) [inline pkg/metaserver/agent/metric/metric.go:617]: It's not a good idea to use comments to delete code.

Comment (sun-yuliang) [reply pkg/metaserver/agent/metric/metric.go]: fixed

**Review by waynepeking348 — APPROVED** [review]

**Pull request merged**